mainnet_contract/contracts/LiteCheckpoint.sol

```diff
@@ -25,7 +25,9 @@ contract LiteCheckpoint {
         address[] set;
         int256 threshold;
     }
-    mapping(bytes32 => bytes32) private unCommittedLastHash;
+    //record uncommit gappushone/epoch block commit last hash 
+    //for example received 451 452 453 , the last hash is 453'hash
```

Comment:
gappushone -> gapPlusOne.

for example received 451 452 453 , the last hash is 453'hash -> "the map is 451 hash => 453 hash"

Reply:
fixed